package-lock.json

```diff
@@ -9552,6 +9553,7 @@
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/delayed-stream/-/delayed-stream-1.0.0.tgz",
       "integrity": "sha512-ZySD7Nf91aLB0RxL4KGrKHBXl7Eds1DAmEdcoVawXnLD7SDhpNgtuII2aAkg7a7QS41jxPSZ17p4VdGnMHk3MQ==",
+      "dev": true,
```

Comment:
this is added by npm, which signifies that this package is now a devDependency. It could have been that form-data had this steams package as required in its package. but since now its removed but `jest-cli -> form-data` has this as devDependency. Hence the changes.